rs/types/types/src/messages/inter_canister.rs

```diff
@@ -612,10 +612,11 @@ impl From<Response> for RequestOrResponse {
 impl From<&RequestMetadata> for pb_queues::RequestMetadata {
     fn from(metadata: &RequestMetadata) -> Self {
         Self {
-            call_tree_depth: Some(metadata.call_tree_depth),
-            call_tree_start_time_nanos: Some(
-                metadata.call_tree_start_time.as_nanos_since_unix_epoch(),
-            ),
+            // TODO(MR-641): Remove deprecated fields.
+            deprecated_opt_call_tree_depth: None,
+            deprecated_opt_call_tree_start_time_nanos: None,
```

Comment:
Also, this is not backwards-compatible. If we rolled back a replica running this code, the old replica version would decode all `RequestMetadatas` encoded this way as `(0, 0)`.